dali/operators/image/paste/paste.cc

```diff
@@ -17,28 +17,29 @@
 namespace dali {
 
 DALI_SCHEMA(Paste)
-  .DocStr(R"code(Paste the input image on a larger canvas.
-The canvas size is equal to `input size * ratio`.)code")
+  .DocStr(R"code(Paste the input image on a larger canvas, and the canvas size is equal to
+the ``input size * ratio``.)code")
   .NumInput(1)
   .NumOutput(1)
   .AddArg("ratio",
-      R"code(Ratio of canvas size to input size, must be > 1.)code",
+      R"code(Ratio of canvas size to input size, which must be greater than 1.)code",
```

Comment:
```suggestion
      R"code(Ratio of canvas size to input size. Must be at least 1.)code",
```
1. >= 1, in fact
2. I think that putting this "which" here makes it look a bit strange...